### PR TITLE
Adding board support for CC3220sf

### PR DESCRIPTION
**Pull request by dragoniteArm**

### CODE REVIEW

File: pyOCD/board/board_ids.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def __init__(self, name, target, binary):
     "1600": BoardInfo(  "Bambino 210",          "lpc4330",          "l1_lpc4330.bin",       ),
     "1605": BoardInfo(  "Bambino 210E",         "lpc4330",          "l1_lpc4330.bin",       ),
     "2201": BoardInfo(  "WIZwik_W7500",         "w7500",            "l1_w7500mbed.bin",     ),
+    "3300": BoardInfo(  "CC3220SF",             "cc3220sf",         "l1_cc3220sf.bin",     ),
```
Comment (flit) [inline pyOCD/board/board_ids.py]: Looks like the `l1_cc3220sf.bin` binary is missing from the changeset?

Comment (dragoniteArm) [reply pyOCD/board/board_ids.py]: Sorry, I just pushed the test binary. Do I need to resubmit a pull request?

Comment (flit) [reply pyOCD/board/board_ids.py]: No, it adds to this PR.

**Comment by flit** [comment]
Since you did a merge, could you please rebase and squash, then force push to update this PR? That will keep the history cleaner. Thanks.

**Pull request merged**